scss/components/_tooltips.scss

```diff
@@ -0,0 +1,88 @@
+//tooltip position
+$ico-size: 23px; //get from _xsmall.scss
+$ico-size-small: 25px * $xsmall-multiple; //calc get from _xsmall.scss
+$items-wrapper-margin-top: 10px;
+$tooltip-offset: $ico-size + $items-wrapper-margin-top + ($scrubber-bar-height/2 + $scrubber-bar-padding-height/2);
+$tooltip-offset-small: $ico-size-small + $items-wrapper-margin-top + ($scrubber-bar-height-small/2 + $scrubber-bar-padding-height-small/2);
+
+//tooltip
+$tooltip-opacity: 0.75;
+$tooltip-color: black;
+$tooltip-animation-duration: 0.5s;
+$tooltip-pointer-size: 5px;
+
+.oo-tooltip::before, .oo-tooltip::after{
+  position: absolute;
+  z-index: $zindex-popover;
+  white-space: nowrap;
+  opacity: 0;
+}
+
+.oo-tooltip::before {
+  content: attr(data-tooltip) ;
+  font-family: $font-family-roboto-condensed;
+  text-shadow: none;
+  background: $tooltip-color;
+  border-radius: 3px;
+  font-size: 15px;
+  padding: 8px 20px 8px 20px;
+  bottom: $tooltip-offset + $tooltip-pointer-size;
+}
+
+.oo-xsmall .oo-tooltip::before {
```

Comment:
It looks like we have all the `oo-small`, etc. styles in a section of the corresponding size file. I'm not a big fan of this approach, but it might make sense to move these styles there for consistency.